docs/usage/observability/logging.md

```diff
@@ -53,6 +53,7 @@ The logs are accessible via Plutono. To access them:
   1. Authenticate via basic auth to gain access to Plutono.
   The Plutono URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside the credentials.
```

Comment:
>   The Plutono URL can be found in the `Logging and Monitoring` section of a cluster in the Gardener Dashboard alongside the credentials.

IIUC, https://github.com/gardener/gardener/issues/9867 suggest dropping the Gardener Dashboard part here. Because I assume after the oidc-apps-controller introduction, the dashboard now shows the OIDC URLs. @petersutter @nickytd to confirm/reject.

Reply:
yes, the dashboard shows the OIDC URLs if enabled and otherwise falls back to the basic auth urls